docs/.ipynb_checkpoints/MaskMaker_overview-checkpoint.ipynb

```diff
@@ -0,0 +1,808 @@
+{
```

Comment:
we should add a line in the .gitignore to ignore these backup files so they don't clutter the commit history/ storage space